src/scene/export.ts

```diff
@@ -39,29 +43,59 @@ export function exportToCanvas(
 
   const tempCanvas: any = createCanvas(width, height);
 
+  const sceneState = {
+    viewBackgroundColor: exportBackground ? viewBackgroundColor : null,
+    scrollX: normalizeScroll(-minX + exportPadding),
+    scrollY: normalizeScroll(-minY + exportPadding),
+    zoom: 1,
+    remotePointerViewportCoords: {},
+    remoteSelectedElementIds: {},
+    shouldCacheIgnoreZoom: false,
+    remotePointerUsernames: {},
+  };
+
   renderScene(
     elements,
     appState,
     null,
     scale,
     rough.canvas(tempCanvas),
     tempCanvas,
-    {
-      viewBackgroundColor: exportBackground ? viewBackgroundColor : null,
-      scrollX: normalizeScroll(-minX + exportPadding),
-      scrollY: normalizeScroll(-minY + exportPadding),
-      zoom: 1,
-      remotePointerViewportCoords: {},
-      remoteSelectedElementIds: {},
-      shouldCacheIgnoreZoom: false,
-      remotePointerUsernames: {},
-    },
+    sceneState,
     {
       renderScrollbars: false,
       renderSelection: false,
       renderOptimizations: false,
     },
   );
+
+  if (addWatermark) {
+    const context = tempCanvas.getContext("2d")!;
+    const text = "Made in Excalidraw";
```

Comment:
Let's translate that as well.

Reply:
+1, i think adding i18n would block landing this PR

Reply:
not really, translations are added on the go, they don't block the PR